### ISSUE TITLE
Columns Explorer: Control+NvDA+Number row fails if column count is less than 10

### ISSUE THREAD

**Issue by josephsl** [josephsl/stationPlaylist#115]
Hi,

Related to #114: if a track item column count is less than 10, Columns Explorer does not work properly - instead it announces titles of the last columns that may not exist.

Thus resolve this by:

1. Detect if column count is less than 10 and announce an error message if a column that is out of range is sought (e.g. if column count is 8, pressing Control+NVDA+number row 9 should announce an error message).
2. If a list uses different column headers other than those encountered in tracks list (at least existing classes), fetch column header from ones displayed on the screen (if possible).

Note: this issue must be resolved before #114 can be worked on.

Thanks.
